pdm/pep582/sitecustomize.py

```diff
@@ -1,10 +1,12 @@
 import os
 import site
 import sys
+from pathlib import Path
+from typing import Union
 
 
-def get_pypackages_path(maxdepth=5):
-    def find_pypackage(path, version):
+def get_pypackages_path(maxdepth: int = 5) -> Path:
+    def find_pypackage(path: Path, version: str) -> Union[None, Path]:
```

Comment:
Don't type this file, it can be run by Python 2.7

Reply:
Fixed here. Sorry I didn't notice it.

[`312287c` (#354)](https://github.com/pdm-project/pdm/pull/354/commits/312287c375db739caa180500afbc976002515113#diff-e5d05cfeca8df090e45a786f1e58cac1d4fc31c17897dcb8d37bae44ababf47fR6-R7)